test/unit/css.js

```diff
@@ -664,6 +654,190 @@ QUnit.test( "show() after hide() should always set display to initial value (#14
 	assert.equal( div.css( "display" ), "list-item", "should get last set display value" );
 } );
 
+QUnit.test( "show/hide 3.0, default display", function( assert ) {
```

Comment:
Do you have a suggestion? The assertions really are dependent upon methods that have already been called.